domains/_discord.yurei.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "yurehito
```

You forgot closing quotes and comma

```suggestion
        "username": "yurehito",
```